recommendation/java/vertx/src/main/java/com/redhat/developer/demos/recommendation/RecommendationVerticle.java

```diff
@@ -9,13 +9,17 @@
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 
-
 public class RecommendationVerticle extends AbstractVerticle {
 
     private static final String RESPONSE_STRING_FORMAT = "recommendation v1 from '%s': %d\n";
 
-    private static final String HOSTNAME =
-        parseContainerIdFromHostname(System.getenv().getOrDefault("HOSTNAME", "unknown"));
+    private static final String HOSTNAME = parseContainerIdFromHostname(
+        System.getenv().getOrDefault("HOSTNAME", "unknown")
+    );
+
+    private static final int RECOMMENDATION_PORT = Integer.parseInt(
+        System.getenv().getOrDefault("RECOMMENDATION_PORT", "8080")
```

Comment:
This env var is actually provided by OpenShift, in the format `tcp://172.30.229.253:8080`, so, this will cause an exception:

```
Caused by: java.lang.NumberFormatException: For input string: "tcp://172.30.229.253:8080"
```